html-syntax/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <img src="https://cdn.bulbagarden.net/upload/6/67/658Greninja.png" alt="Geninja" width="600" height="600" />
+  <h1><b>Greninja</b></h1>
+  <h2>&copy; Pokémon. &reg; Nintendo.</h2>
+  </body>
+</html>
```

Comment:
This is almost done, however when creating an HTML Skeleton, you want to remember to add head tags with a title tag.
You can review the previous exercise's reading

https://lfz-lms.herokuapp.com/code-exercises/Learning-Fuze/curriculum/html-skeleton?v=1363c0bdfe575ddb063022e0962d7a7ea314df53